core/include/mm/vm.h

```diff
@@ -87,26 +83,20 @@ TEE_Result vm_buf_to_mboj_offs(const struct user_mode_ctx *uctx,
 			       const void *va, size_t size,
 			       struct mobj **mobj, size_t *offs);
 
-/*-----------------------------------------------------------------------------
- * vm_va2pa - Translate virtual user address to physical address
- * given the user context.
- * Interface is deprecated, use virt_to_phys() instead.
- *---------------------------------------------------------------------------*/
+/* This function is far less efficient than virt_to_phys() */
 TEE_Result vm_va2pa(const struct user_mode_ctx *uctx, void *ua, paddr_t *pa);
 
-/*-----------------------------------------------------------------------------
- * vm_pa2va - Translate physical address to virtual user address
- * given the user context.
- * Interface is deprecated, use phys_to_virt() instead.
- *---------------------------------------------------------------------------*/
-TEE_Result vm_pa2va(const struct user_mode_ctx *uctx, paddr_t pa, void **va);
+/* This function may be less efficient than phys_to_virt() */
```

Comment:
Instead of this comment it could be worth noting that this is a helper function for phys_to_virt() and shouldn't be used directly elsewhere.